src/commands/utils/vatsimData.ts

```diff
@@ -0,0 +1,233 @@
+            if ((Object.keys(vatsimControllers).length !== 0 || Object.keys(vatsimAtis).length !== 0) && (commandMode === 'ALL' || commandMode === 'CONTROLLERS')) {
+                const fields: EmbedField[] = [...vatsimControllers.sort((a, b) => b.facility - a.facility), ...vatsimAtis].map((vatsimItem) => {
+                    // eslint-disable-next-line camelcase
+                    const { callsign, frequency, logon_time, atis_code, text_atis, rating } = vatsimItem;
+                    const logonTime = new Date(logon_time);
+                    const logonTimeString = handleLocaleDateTimeString(logonTime);
+                    const ratingDetail = vatsimControllerRatings.filter((ratingInfo) => ratingInfo.id === rating);
+                    const ratingText = `${ratingDetail[0].short} - ${ratingDetail[0].long}`;
+                    const data = [
+                        {
+                            name: 'Callsign',
+                            value: `${callsign}`,
+                            inline: false,
+                        },
+                        {
+                            name: 'Frequency',
+                            value: `${frequency}`,
+                            inline: true,
+                        },
+                        {
+                            name: 'Logon Date & Time',
+                            value: `${logonTimeString}`,
+                            inline: true,
+                        },
+                        {
+                            name: 'Rating',
+                            value: `${ratingText}`,
+                            inline: true,
+                        },
+                    ];
+
+                    // eslint-disable-next-line camelcase
+                    if (text_atis !== null) {
+                        // eslint-disable-next-line camelcase
+                        const atisTitle = atis_code ? `ATIS - Code: ${atis_code}` : 'ATIS';
+                        data.push({
+                            name: `${atisTitle}`,
+                            // eslint-disable-next-line camelcase
+                            value: `${text_atis}`,
+                            inline: false,
+                        });
+                    }
+
+                    return data;
+                }).slice(0, 5).flat();
+
+                const totalCount = Object.keys(vatsimControllers).length + Object.keys(vatsimAtis).length;
+                const shownCount = totalCount <= 5 ? totalCount : 5;
+                responseEmbed = makeEmbed({
+                    title: `VATSIM Data | ${searchTerm} - ${totalCount} Controllers, Observers & ATIS online`,
+                    description: `A list of ${shownCount} online VATSIM Controllers, Observers & ATIS matching ${searchTerm}`,
+                    fields,
+                });
+            } else if (Object.keys(vatsimPilots).length !== 0 && (commandMode === 'ALL' || commandMode === 'PILOTS')) {
```

Comment:
This is being a bit tedious but this is being used more than once, maybe make it a const so you can reuse it?
```ts
Object.keys(vatsimPilots).length
```